modin/tests/pandas/dataframe/test_default.py

```diff
@@ -87,7 +88,11 @@
 )
 def test_ops_defaulting_to_pandas(op, make_args):
     modin_df = pd.DataFrame(test_data_diff_dtype).drop(["str_col", "bool_col"], axis=1)
-    with warns_that_defaulting_to_pandas():
+    with (
+        warns_that_defaulting_to_pandas()
```

Comment:
It's better to update `warns_that_defaulting_to_pandas` itself to work with `NativeDataframeMode`.

Reply:
Updated warns_that_defaulting_to_pandas to return nullcontext in case of NativeDataframeMode